v2/models/event.go

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package models
+
+// Event represents a single measurable event read from a device
+type Event struct {
+	CorrelationId string
+	Checksum      string
+	ID            string    `json:"id,omitempty" codec:"id,omitempty"`             // ID uniquely identifies an event, for example a UUID
```

Comment:
As these are internal models, why do they need to be annotated?

Reply:
Removed the `json` and `codec` tags for models.